packages/web-components/src/mixins/uswds-error-border.scss

```diff
@@ -2,7 +2,7 @@
 @import 'functions';
 
 :host([error]:not([error=''])[uswds]:not([uswds='false'])) {
-    border-left: 0.4rem solid $v3-color-error-dark; /* USWDS red-60v, $theme-color-error-dark */
+    border-left: 0.4rem solid $color-secondary-dark; /* USWDS red-60v, $theme-color-error-dark */
```

Comment:
Nice! Just a very small nitpick but now that this is updated, can we remove the comment next to it? It doesn't quite make sense any more.